projects/fsastorefrontlib/src/cms-components/comparison-table/comparison-table-container/comparison-table-container.component.ts

```diff
@@ -18,32 +21,61 @@ import { ComparisonTableService } from '../comparison-table.service';
 export class ComparisonTableContainerComponent implements OnInit, OnDestroy {
   constructor(
     protected componentData: CmsComponentData<CmsMultiComparisonTabContainer>,
-    protected comparisonTableService: ComparisonTableService
+    protected cmsService: CmsService
   ) {}
 
   component$: Observable<CmsMultiComparisonTabContainer>;
-  tabs$;
+  tabs$: Observable<ComparisonPanelCMSComponent>;
   active = 0;
+  initialTabs: string[];
+  availableTabs = [];
 
   private subscription = new Subscription();
 
   ngOnInit() {
     this.component$ = this.componentData.data$;
+    this.getInitialTabs();
+    this.getAvailableTabs();
+  }
+
+  /**
+   * Fetches all tabs available from BE configuration
+   */
+  getInitialTabs() {
     this.subscription.add(
       this.component$
         .pipe(
           map(data => {
             if (data.simpleCMSComponents) {
```

Comment:
Maybe you can consider to use `filter` operator (`filter(data) => !!data. simpleCMSComponents`), instead of `if` condition..